kousa/lib/kousa/room.ex

```diff
@@ -165,6 +165,23 @@ defmodule Kousa.Room do
       )
     end
   end
+	
+  def change_room_creator(new_creator_id, current_room_id \\ nil) do
```

Comment:
- reset `package-lock.json`
- also need to change room creator in the DB
- backend validation: 
    - check if the user who initiated the request is the room creator of the room which they're in
    - check if the user_to_be_new_creator is in the same room as the room creator

Reply:
no problem

Reply:
@jamesql I appreciate it

Reply:
@nadirabbas take a look at it now, let me know if anything else should be changed